src/pages/DonateFiatThanks/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { Link } from "react-router-dom";
+import Icon from "components/Icon";
+import { IS_AST } from "constants/env";
+import { appRoutes } from "constants/routes";
+
+export default function DonateFiatThanks() {
+  return (
+    <div className="justify-self-center display-block m-auto max-w-[35rem] py-8 sm:py-20 scroll-mt-6">
+      <Icon type="CheckCircle" size={96} className="text-green mb-4 ml-10" />
```

Comment:
Can we center this icon over the text please? I think this may have been my bad from when I moved it into a page and reduced some of the wrappers of divs around the contents. Everything else on the page looks OK. 

![image](https://user-images.githubusercontent.com/85138450/233312492-a72de08d-ef5f-4ffe-87aa-076f73f10a16.png)

Reply:
Done